src/ensembl/production/metadata/api/models/genome.py

```diff
@@ -54,18 +54,41 @@ def get_public_path(self, type='all', release=None):
 
         genebuild_source_name = genebuild_annotation_source_attribute.value
         common_path = f"{self.organism.scientific_name.replace(' ', '_')}/{self.assembly.accession}/{genebuild_source_name}"
-
-        if type in ['genebuild', 'assembly', 'homology', 'regulation', 'variation', 'all']:
+        unique_dataset_types = {gd.dataset.dataset_type.name for gd in self.genome_datasets}
+
+        standard_types = {'genebuild', 'assembly', 'homologies', 'regulation_build', 'regulatory_features', 'variation'}
```

Comment:
We could fetch from dataset_type table, or use class Constants instead, so if used elsewhere and change overtime, we'll have to change it in one place.